users/templates/registration/login.html

```diff
@@ -30,19 +31,6 @@
         <span><a class="underlineHover" href="{% url 'password_reset' %}">Forgot your password?</a></span>
     </div>
   </div>
-     {% if form.errors %}
```

Comment:
It seems that those errors are missing - only `form.non_field_errors` seem to be handled in changed code.